tests/__snapshots__/test_help_messages.ambr

```diff
@@ -6447,6 +6533,7 @@
   | create           Creates a new service in the current schema.                |
   | describe         Provides description of service.                            |
   | drop             Drops service with given name.                              |
+  | execute-job      Execute a job service in the current schema.                |
```

Comment:
nit: `Executes` to be consistent with other descriptions.

Do you think it would be helpful if we say
```
Creates and executes a job in the current schema.
```
to highlight that this also acts as a creation operation. Not mentioning "job service" is just for my own personal preference.

Reply:
Make sense. The job command is in the service command group so mentioning `job service` to avoid confusion.